addons/web/static/src/core/ui/ui_service.js

```diff
@@ -25,12 +25,12 @@ function getFirstAndLastTabableElements(el) {
  *
  * @param {string} refName
  */
-export function useActiveElement(refName) {
-    if (!refName) {
+export function useActiveElement(refName, refObj) {
+    if (!refName && !refObj) {
         throw new Error("refName not given to useActiveElement");
     }
     const uiService = useService("ui");
-    const ref = useRef(refName);
+    const ref = refObj || useRef(refName);
```

Comment:
Having the first argument being ignored when we use the second one is a bit awkward, I think something like this would be cleaner:
```js
export function useActiveElement(ref) {
    if (!ref) {
        throw new Error("ref not given to useActiveElement");
    }
    const uiService = useService("ui");
    ref = typeof ref === "string" ? useRef(ref) : ref;
```

Reply:
yes, I'm definitely not sure about those changes.... :D

Reply:
what do you thin @bastien-ipb  ?